recipe/meta.yaml

```diff
@@ -9,32 +9,31 @@ source:
   sha256: fc024ed53c7ea090e30723e6bd82f58a39dc25d9a6797d866203ecd0ee6306cb
 
 build:
-  number: 0
-  skip: true  # [py<38]
+  number: 1
+  noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - keyring = keyring.cli:main
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.8
     - pip
     - setuptools_scm >=3.4.1
     - setuptools >=56
     - toml
   run:
-    - python
+    - python >=3.8
     - pywin32-ctypes >=0.2.0  # [win]
     - secretstorage >=3.2  # [linux]
     - jeepney >=0.4.2  # [linux]
-    - importlib_metadata >=4.11.4  # [py<312]
+    - importlib_metadata >=4.11.4
     - jaraco.classes
-    - importlib_resources  # [py<39]
+    - importlib_resources
     - jaraco.functools
     - jaraco.context
+    - __linux  # [linux]
```

Comment:
don't we have a 'unix'? you will need something for osx.

Reply:
I don't understand very well your point. The `secretstorage` and `jeepney` deps are Linux specific (i.e. they are not required for osx). So, do I need to add

    - __osx # [osx]

too so that we can have a package for Mac?

Reply:
correct, presently you are building a:

1. Linux variant
2. Windows variant

but there is nothing that can be installed for OSX.

So you need "something" there, either a 3rd variant, or relaxing the linux -> unix constraint. It seems that think (and you are probably right) that a different variant is needed.